distribution/test/itests/pom.xml

```diff
@@ -78,7 +80,7 @@
             <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
-                <version>4.11</version>
+                <version>${junit.version}</version>
```

Comment:
We were thinking the same thing, when we tried removing it ourselves it broke. Something isn't correct in our pom structure. I don't know if we need separate ticket to investigate this.